frontend/server/src/Controllers/Problem.php

```diff
@@ -5695,4 +5696,58 @@ public static function getLibinteractiveGenForSmarty(\OmegaUp\Request $r): array
             \OmegaUp\FileHandler::deleteDirRecursively($dirname);
         }
     }
+
+    /**
+     *
+     * @return array{smartyProperties: array{payload: ProblemCollectionDetails, title: \OmegaUp\TranslationString}, entrypoint: string}
+     *
+     * @omegaup-request-param string $collection_type
+     */
+    public static function getCollectionsDetailsForSmarty(\OmegaUp\Request $r): array {
+        $collectionType = $r->ensureString('collection_type');
+
+        $collection = null;
+
+        $title = new \OmegaUp\TranslationString(
+            'omegaupTitleCollectionsByLevel'
+        );
+
+        if ($r->ensureString('collection_type') !== 'author') {
+            $collection = \OmegaUp\Controllers\Tag::getFrequentTagsByLevel(
+                $r->ensureString('collection_type')
+            );
+        } else {
+            //Author
```

Comment:
nit:

```suggestion
            // Author
```

todos los demás comentarios llevan un espacio después del `//` (tengo que ver si el linter puede arreglar esto automágicamente).